lib/travis/api/v3/proxy_service.rb

```diff
@@ -30,7 +30,7 @@ def proxy!
     class ProxyClient
       def initialize(endpoint, auth_token)
         @connection = Faraday::Connection.new(URI(endpoint)) do |conn|
-          conn.headers[:Authorization] = "Token token=\"#{auth_token}\""
+          conn.token_auth auth_token
```

Comment:
this is faraday related, keep the master + update faraday back to 2

Reply:
Done.